packages/validation/test/parseValues/parseField.spec.ts

```diff
@@ -300,5 +300,34 @@ describe('Parse Values - parseFieldValue', () => {
 			it('Boolean array field, rejects array where value is missing (two delimiters are adjacent)');
 			expect(parseFieldValue(',true,false,TRUE', fieldBooleanArrayRequired).success).false;
 		});
+		describe('Custom delimiter', () => {
```

Comment:
nice feature! "out loud" thoughts about this, not to question your coverage but to narrate my analysis: asking myself "how do you escape the delimiter if it's also meant to be used as a value?" like you would in a comprehensive array of symbols. the answer I'd guess would be "use a compound delimiter", and then I realised it's not clear what happens if I give the delimiter repeatedly. does the array end up with an empty value? or does the function filter empties?

e.g. assume default (i.e. `,`), what if I give you `symbols = ',",\,%,,,#,*` do I end up with `[' , " , \ , % , , , # , *].length === 8`? or `[' , " , \ , % , # , *].length === 6`
... and what would be the representation of the quotes in JS, since one of the two first elements here would have to be escaped to make a string.
There's special handling here that could use being documented, methinks?

Reply:
if you put the delimiter back-to-back this results in an empty value between them. This is not allowed in arrays where each element is expected to have a value. Validation of the data will fail.

https://github.com/overture-stack/lectern/blob/1c32ea4b15233050f2e2383f7b107e4477874cb8/packages/validation/test/parseValues/parseField.spec.ts#L269-L271

Reply:
There is no handling for escaping characters. If you are providing data in a TSV, all characters are allowed. If you are 
providing data in JSON, then the JSON parsing will strip escape characters.